### ISSUE TITLE
Support ARG instructions before FROM

### ISSUE THREAD

**Issue by rcjsuen** [projectatomic/dockerfile_lint#85]
As reported in [Microsoft/vscode-docker#120](https://github.com/Microsoft/vscode-docker/issues/120), `ARG` instructions can now appear before the first `FROM` instruction so this shouldn't be flagged as an error by dockerfile_lint.

```Dockerfile
ARG CODE_VERSION=5.6-apache
FROM php:$CODE_VERSION
```
```
$ node bin\dockerfile_lint -f Dockerfile

--------ERRORS---------

Line 1: -> ARG CODE_VERSION=5.6-apache
ERROR: Missing or misplaced FROM.
Reference -> https://docs.docker.com/engine/reference/builder/


Line 2: -> FROM php:$CODE_VERSION
ERROR: Invalid parameters for command..
Reference -> https://docs.docker.com/engine/reference/builder/


ERROR: Required LABEL name/key 'Name' is not defined.
Reference -> http://docs.projectatomic.io/container-best-practices/#_recommended_labels_for_your_project


ERROR: Required LABEL name/key 'Version' is not defined.
Reference -> http://docs.projectatomic.io/container-best-practices/#_recommended_labels_for_your_project



--------INFO---------

INFO: There is no 'EXPOSE' instruction. Without exposed ports how will the service of the container be accessed?.
Reference -> https://docs.docker.com/engine/reference/builder/#expose


INFO: There is no 'CMD' instruction. None.
Reference -> https://docs.docker.com/engine/reference/builder/#cmd
```

**Comment by alexandruivan** [comment]
Hello, was this released yet?

**Comment by alexandruivan** [comment]
I created a pull request for this: https://github.com/projectatomic/dockerfile_lint/pull/88

**Comment by James1912** [comment]
Has this gotten released yet? Would be lush if could get published

**Comment by lphiri** [comment]
@James1912 @alexandruivan  included in 0.3.0

**Comment by lphiri** [comment]
Fixed in current release
